plugins/arDominionB5Plugin/js/collapse.js

```diff
@@ -6,10 +6,10 @@
     if ($('#editForm .accordion-item').length) {
       var $collapseToShow = $(location.hash);
       if ($collapseToShow.length) {
-        $collapseToShow.on('shown.bs.collapse', function (e) {
+        $collapseToShow.on('shown.bs.collapse', (e) => {
           window.scrollTo(0, $(e.target).parent().offset().top);
         });
-        bootstrap.Collapse.getOrCreateInstance($collapseToShow.get(0)).show();
+        bootstrap.Collapse.getOrCreateInstance($collapseToShow).show();
```

Comment:
I think the show is not needed as the getOrCreateInstance call toggles the collapse by default.

Reply:
Nice. Thanks!